models/Issue.js

```diff
@@ -0,0 +1,58 @@
+const { Schema, model } = require("mongoose");
+
+const IssueSchema = new Schema({
+  name: {
+    type: String,
+    required: true,
+    minlength: 5,
+  },
+  description: {
+    type: String,
+    default: "",
+  },
+  section: {
+    type: String,
+    required: true,
+  },
+  department: {
+    type: String,
+    required: true,
+  },
+  isEdited: {
+    type: Boolean,
+    default: false,
+  },
+  isResolved: {
+    type: Boolean,
+    default: false,
+  },
+  isInappropriate: {
+    type: Boolean,
+    default: false,
+  },
+  isDeleted: {
+    type: Boolean,
+    default: false,
+  },
+  upvotes: {
+    type: Number,
+  },
+  upvoters: {
+    type: [Number],
```

Comment:
I guess this is a list of all the `zprn` that have upvoted the issue right?

Reply:
Yes, sir.